VMware.CloudFoundation.Reporting.psm1

```diff
@@ -1145,6 +1145,91 @@ Function Publish-ConnectivityHealth {
 }
 Export-ModuleMember -Function Publish-ConnectivityHealth
 
+Function Publish-PingConnectivityHealth {
+    <#
+        .SYNOPSIS
+        Formats the PingConnectivity Health data from the SoS JSON output.
```

Comment:
```suggestion
        Formats the Ping Connectivity Health data from the SoS JSON output.
```

Reply:
Resolved in 343e932d8ec8e2e0039d3a5ba19c568a5ef72754.